Cargo.toml

```diff
@@ -111,6 +112,9 @@ vlog = "0.1.4"
 mime = "0.3.13"
 farmhash = "1.1.5"
 failure = "0.1"
+pprof = { version = "0.3", features = ["flamegraph", "protobuf"] }
+prost = "0.5.0"
```

Comment:
oh, we have to depend on prost now
/cc @nrc

Reply:
Why do we have to depend on Prost? I would think anything that is done with Prost can be done with rust-proto